src/OpenApi/test/Microsoft.AspNetCore.OpenApi.Tests/Services/OpenApiDocumentService/OpenApiDocumentServiceTests.Responses.cs

```diff
@@ -440,4 +440,104 @@ await VerifyOpenApiDocument(builder, document =>
                 });
         });
     }
+
+    [Fact]
+    public async Task GetOpenApiResponse_MinimalApi_MultipleProducesSameStatusCode_DifferentContentTypes()
+    {
+        // Arrange
+        var builder = CreateBuilder();
+
+        // Act
+        builder.MapGet("/api/todos", () => { })
+            .Produces(StatusCodes.Status200OK, typeof(Todo), "application/json")
+            .Produces(StatusCodes.Status200OK, typeof(Todo), "application/xml");
+
+        // Assert
+        await VerifyOpenApiDocument(builder, document =>
+        {
+            var operation = Assert.Single(document.Paths["/api/todos"].Operations.Values);
+            var response = Assert.Single(operation.Responses);
+            Assert.Equal("200", response.Key);
+            Assert.Equal("OK", response.Value.Description);
+            // Note: Our PR implementation is still in progress
```

Comment:
This is not correct. Remove this comment and add tests that validate that there are two content types under the response.